tofu/geom/_core.py

```diff
@@ -5969,7 +5971,7 @@ def calc_signal(
         self,
         func,
         t=None,
-        ani=None,
```

Comment:
Same comment as above, None means the user did not specify it, False means he wants to force it to False

Reply:
updated